Dockerfile.vcsim

```diff
@@ -1,4 +1,45 @@
+# Create a builder container
```

Comment:
Can we please specify which container tag/arch this references so it's more clear when upgrading, like in the example I gave?

Reply:
Yes, sorry, I misunderstood that.